src/main/java/seedu/address/model/client/Client.java

```diff
@@ -13,6 +13,8 @@
  */
 public class Client {
 
+    public static final String MESSAGE_INVALID = "Client does not exist in the project.";
```

Comment:
I am not sure if this ought to be here considering that in our implementation, a `Client` should be able to exist without a `Project`. So, perhaps we could place this message  in some other class dealing with messages or rephrase it?